cmake/FindLibBcc.cmake

```diff
@@ -38,9 +38,9 @@ find_library (LIBBCC_LIBRARIES
     ENV LIBRARY_PATH
     ENV LD_LIBRARY_PATH)
 
-find_library (LIBBPF_LIBRARY_STATIC
+find_library (LIBBCC_BPF_LIBRARY_STATIC
   NAMES
-    bpf
+    bccbpf
```

Comment:
I was discussing with @danobi that it is confusing we have lib_bpf from both FindLibBcc and findLibBpf. I think this is because lib_bpf started being vendored by bcc before it was merged into the kernel tree.

I don't think it's within the scope of what this PR needs to worry about, but i thought it was worth pointing out the quirk.